spec/serpapi/search_archive_api_spec.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'spec_helper'))
+
+describe 'SerpApi Search Archive API' do
+  it 'fetch a search result from the archive' do
+    # mock
+    search_response_mock = '{
+        "search_metadata": {
+          "id": "cdbdbdebeabab1beb"
+        }
+      }'
+    client = SerpApi::Client.new(api_key: ENV['API_KEY'], engine: 'google')
+    if client.api_key.nil?
+      allow(search).to receive(:get_results) { search_response_mock }
```

Comment:
Is `search` defined?

Reply:
good catch. forgot to rename search to client